src/finance_quote/modules/fixerio.py

```diff
@@ -0,0 +1,185 @@
+"""
+Fixer.io currency rates 
+Currently only implements the free public API access with yesterday's rates.
+"""
+import os
+import tempfile
+from datetime import datetime
+import logging
+from decimal import Decimal
+
+from finance_quote.base import Source, Quote
+import finance_quote
+
+try: import simplejson as json
+except ImportError: import json
+
+
+class FixerioQuote(Quote):
+    """ Fixer.io-specific quote """
+    pass
+
+
+class Fixerio(Source):
+    """Retrieves prices from data files or online provider(s)"""
+    def __init__(self):
+        self.cache_path = tempfile.gettempdir()
+        self.logger = logging.getLogger(__name__)
```

Comment:
logger is now initialised in `Source.__init__`

Reply:
+1

Reply:
I don't see that in the code so it's a catch-22 situation. Can't merge that change and can't remove this one because then the execution fails.